pkg/apis/kops/componentconfig.go

```diff
@@ -823,10 +823,10 @@ type OpenstackConfiguration struct {
 	Metadata           *OpenstackMetadata           `json:"metadata,omitempty"`
 }
 
-// AzureConfiguration defines Azure specific cluster configuration.
-type AzureConfiguration struct {
+// AzureSpec defines Azure specific cluster configuration.
+type AzureSpec struct {
 	// SubscriptionID specifies the subscription used for the cluster installation.
-	SubscriptionID string `json:"subscriptionId,omitempty"`
+	SubscriptionID string `json:"subscriptionID,omitempty"`
```

Comment:
Aside: Are we actually using this JSON encoding anywhere?  I know there are some places e.g. in nodeup where we aren't we precise with the version, but this is in the unversioned form...

Reply:
I'm not sure. I think we keep it in sync with the latest versioned API just to make updating easier.